imjoy/const.py

```diff
@@ -2,8 +2,9 @@
 import json
 import pathlib
 
-DEFAULT_REQUIREMENTS_PY2 = ["requests", "six", "websocket-client", "numpy"]
-DEFAULT_REQUIREMENTS_PY3 = [*DEFAULT_REQUIREMENTS_PY2, "janus"]
+DEFAULT_REQUIREMENTS = ["numpy"]
+DEFAULT_REQUIREMENTS_PY2 = [*DEFAULT_REQUIREMENTS, "python-socketio[client]"]
```

Comment:
How about place `numpy` directly here and remove `DEFAULT_REQUIREMENTS`.

Reply:
Will we ever extend the default requirements beyond `numpy`? In that case I'd prefer to only have to update in one place.

Reply:
I cannot predict exactly, but I think no, we don't need more. We can even think make numpy optional.

Since the `const.py` is more used by the engine, maybe we should even move these requirements to placed it with the runners. 

Maybe have a runner registry, to allow register new runner types, and each runner has a piece of startup code to install dependencies from pip/conda, build docker image etc.

Reply:
These requirements lists are used by the engine to install the correct requirements for the plugin runner. So at the moment they should stay in const.py. I think we can think more about this and keep it as it is for now.

Reply:
Sounds good.

Sound good to me. 

What I meant here is more about `DEFAULT_REQUIREMENTS` couples the two workers for py2 and py3.